src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -132,4 +134,31 @@ public IndexMap getIndexerMap() {
         return new IndexMap();
     }
 
+    public static class LEDMap {
+        public AddressableLED visionLED() {
+            return new AddressableLED(0);
+        }
+
+        public int visionBufferLength() {
+            return 0;
+        }
+
+        public ISolenoid visionRingLight() {
```

Comment:
The lights are a solenoid? Change the name please.

Reply:
Any suggestions for a name?